Automattic/wp-calypso
# Jetpack: Add personal plan

This PR adds the jetpack personal plan to calypso. It's mostly comprised of the plan configuration.

It also adds a feature flag (`jetpack/personaPlan`) open to every environment but production, so we could merge this right away before releasing the plan to everybody.

![image](https://cloud.githubusercontent.com/assets/1554855/20003650/fd148ae2-a287-11e6-93cd-86078125c554.png)

![image](https://cloud.githubusercontent.com/assets/1554855/20003656/058ddc14-a288-11e6-8b9b-7c0fcb767a7d.png)

How to test
=========
0. You need a disconnected jetpack site
1. Connect your site. Once you get to the plans selection page, you should see the personal plan
2. Choose the plan and make sure that you can purchase it without any problem
3. Remove the plan from your site
4. Go to http://calypso.localhost:3000/plans/ and select your site. Make sure that the personal plan is there
5. Purchase the plan again and see everything goes fine

ping @rickybanister @tyxla @richardmuscat @beaulebens
